alembic/versions/13dcbe0a3a64_add_suburban_junction_table.py

```diff
@@ -0,0 +1,37 @@
+"""add suburban junction table
+
+Revision ID: 13dcbe0a3a64
+Revises: ccc480c5d0a2
+Create Date: 2023-01-16 12:39:03.026479
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '13dcbe0a3a64'
+down_revision = 'ccc480c5d0a2'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    op.create_table(
+        'suburban_junction',
+        sa.Column('non_urban_intersection', sa.Integer(), primary_key=True, nullable=False, default=None),
+        sa.Column('non_urban_intersection_hebrew', sa.VARCHAR(length=100)),
```

Comment:
How do we know that the hebrew name of the intersection is limited to 100?

Reply:
We don't... But currently the longest name is around 30. I will truncate the name so it does not cause a crash.